src/modules/MouseUtils/InclusiveMouse/dllmain.cpp

```diff
@@ -0,0 +1,305 @@
+#include "pch.h"
+#include <interface/powertoy_module_interface.h>
+#include <common/SettingsAPI/settings_objects.h>
+#include "trace.h"
+#include "InclusiveCrosshair.h"
+#include "common/utils/color.h"
+
+namespace
+{
+    const wchar_t JSON_KEY_PROPERTIES[] = L"properties";
+    const wchar_t JSON_KEY_VALUE[] = L"value";
+    const wchar_t JSON_KEY_ACTIVATION_SHORTCUT[] = L"activation_shortcut";
+    const wchar_t JSON_KEY_CROSSHAIR_COLOR[] = L"crosshair_color";
+    const wchar_t JSON_KEY_CROSSHAIR_OPACITY[] = L"crosshair_opacity";
+    const wchar_t JSON_KEY_CROSSHAIR_RADIUS[] = L"crosshair_radius";
+    const wchar_t JSON_KEY_CROSSHAIR_THICKNESS[] = L"crosshair_thickness";
+    const wchar_t JSON_KEY_CROSSHAIR_BORDER_COLOR[] = L"crosshair_border_color";
+    const wchar_t JSON_KEY_CROSSHAIR_BORDER_SIZE[] = L"crosshair_border_size";
+}
```

Comment:
Probably, these should be marked as non-localizable in the comment or in the namespace name, as in the other modules.

Reply:
What do you mean?
The other modules that do this don't seem to do that. At least those I checked. Do you have an example?

Reply:
https://github.com/microsoft/PowerToys/blob/714ca349ff76a09de68605f06116934281e0e8d8/src/common/logger/logger_settings.h#L6
https://github.com/microsoft/PowerToys/blob/714ca349ff76a09de68605f06116934281e0e8d8/src/modules/fancyzones/FancyZonesLib/FancyZonesDataTypes.cpp#L8-L19
https://github.com/microsoft/PowerToys/blob/714ca349ff76a09de68605f06116934281e0e8d8/src/modules/powerrename/PowerRenameUIHost/PowerRenameUIHost.cpp#L21
https://github.com/microsoft/PowerToys/blob/714ca349ff76a09de68605f06116934281e0e8d8/src/settings-ui/Settings.UI.Library/UpdatingSettings.cs#L40

Reply:
Thank you!